src/styleguide/docs/principles.md

```diff
@@ -0,0 +1,37 @@
+Design principles are a shared set of guidelines to inform design decisions. Although these are written at a higher level, they are intended to guide the decisions at all levels of implementation, from high-level technical architecture down to copyediting or colour selection.
+
+These principles are an extension of brand values. They are derived from asking: *What does good design mean to us?*
+
+The ordering here is not arbitrary. If the principles are in conflict, prioritise the highest level first— if there's a decision to be made between being bold and being clear, opt for clarity over boldness.
+
+## 1. Be clear.
+
+Design is about creating clarity from chaos. The most effective designs aid comprehension and simplify complex actions, concepts, or interactions.
+
+*What this means in practise:*
+
+Remove unnecessary elements. Use straightforward language. Opt for openness and transparency rather than secrecy. Default to public.
+
+## 2. Be inclusive.
+
+The world has lots of different people in it, all with different abilities, worldviews, cultures, and social perspective. Design and technology do not exist a vacuum.
+
+*What this means in practise:*
+
+Always consider accessibility needs from the outset, rather than tacking it on at the end. Evaluate cultural differences when making visual or functional choices. Include a wide array of different people in user research. Seek to recognise and question your cultural and unconscious biases.
+
+## 3. Be smart.
+
+Technology’s job is to make people’s lives better. It should be intelligent.
+
+*What this means in practise:*
+
+Anticipate users’ needs and meet them. Provide smart, sensible defaults. Make decisions for users, but allow for corrections. Give help, feedback, and prompts based on the users’ current context. Use placeholder text as an opportunity to reinforce best practises.
+
+## 4. Be bold.
+
+Aesthetics are important. It is better to inspire feelings of loathing than apathy. Boldness inspires passion and determination. It's okay to be opinionated so long as you aren't dogmatic.
+
+*What this means in practise:*
+
+Incorporate beauty and playfulness to create an immersive and pleasant user experience. Use brand expression to express moments of exuberance, excitement, passion—signature moments. Make clear decisions based on research and available data, but be willing to course correct if new information becomes available. Pursue the path less travelled.
```

Comment:
I find "course correct" easier to read when hyphenated. I've seen it used both with and without a hyphen in practice, so if it's a case of style I'd definitely advocate for the style with the hyphen. It makes it seem like a single verb, which it kinda is here.

```suggestion
Incorporate beauty and playfulness to create an immersive and pleasant user experience. Use brand expression to express moments of exuberance, excitement, passion—signature moments. Make clear decisions based on research and available data, but be willing to course-correct if new information becomes available. Pursue the path less travelled.
```

Thoughts?

Reply:
![giphy](https://user-images.githubusercontent.com/376315/58595786-bcaad500-8269-11e9-96b6-5e6c38802e21.gif)